nifi-nar-bundles/nifi-mqtt-bundle/nifi-mqtt-processors/src/main/java/org/apache/nifi/processors/mqtt/ConsumeMQTT.java

```diff
@@ -265,7 +266,13 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
         transferQueue(session);
     }
 
-    private void transferQueue(ProcessSession session){
+    // Protected visibility for unit testing
+    protected BlockingQueue<MQTTQueueMessage> getMqttQueue() {
+        return mqttQueue;
+    }
+
+    // Protected visibility for unit testing
+    protected void transferQueue(ProcessSession session){
```

Comment:
Instead of making these protected I would prefer to use reflection to reference them. You can see Oleg and I's discussion on the issue here: https://github.com/apache/nifi/pull/392#discussion_r63536461